apiary.apib

```diff
@@ -5207,6 +5208,9 @@ Translates an ID from the deprecated API into a new UUID.
                 + creditNote
                 + subscription
                 + quotation
+                + timeTracking
+                + deliveryNote
```

Comment:
Don't have any api yet for this, so no need to document it yet.

Reply:
Indeed. I'll remove it.